Vagrantfile

```diff
@@ -24,12 +24,21 @@ SCRIPT
 end
 
 Vagrant.configure("2") do |config|
-  config.vm.box = "boxcutter/ubuntu1610"
   config.vm.network "forwarded_port", guest: 3000, host: 3000
+  # Forward Salt Master ports so that Terraform 
+  # deployed Salt Minions can talk to this VM
+  config.vm.network "forwarded_port", guest: 4505, host: 4505  
+  config.vm.network "forwarded_port", guest: 4506, host: 4506  
+
   config.vm.provision "shell", inline: provision_script
   config.vm.synced_folder ".", "/vagrant"
-  config.vm.provider "virtualbox" do |v|
+  config.vm.provider "virtualbox" do |v, override|
+    config.vm.box = "boxcutter/ubuntu1610"
+    v.memory = 4096
+    v.cpus = 2
+  end
+  config.vm.provider "libvirt" do |v, override|
+    override.vm.box = "yk0/ubuntu-xenial"    
     v.memory = 4096
     v.cpus = 2
   end
-end
```

Comment:
Rubocop is complaining. It should be green to get merged... this `end` shouldn't be removed.